flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/TaskManagerRunner.java

```diff
@@ -349,6 +349,8 @@ public static TaskExecutor startTaskManager(
 		checkNotNull(rpcService);
 		checkNotNull(highAvailabilityServices);
 
+		LOG.info("Starting taskManager {}", resourceID);
```

Comment:
I think "Starting taskManager with ResourceID: {}, resourceID)" would be more explicit. What do you think?

Reply:
Agree @Clarkkkkk , in addition, `taskManager` -> `TaskManager` looks better to me.

Reply:
Agree, thx for your suggestion, have updated the PR according to the comments.